src/Compilers/CSharp/Portable/Binder/RefSafetyAnalysis.cs

```diff
@@ -564,24 +644,90 @@ private void SetPatternLocalScopes(BoundObjectPattern pattern)
             return null;
         }
 
-        private PlaceholderRegion GetArgumentPlaceholders(BoundExpression? receiverOpt, ImmutableArray<BoundExpression> arguments)
+        public override BoundNode? VisitInterpolatedString(BoundInterpolatedString node)
```

Comment:
>VisitInterpolatedString [](http://example.com/codeflow?start=35&length=23)

Interpolated string handler placeholders were not added correctly in all cases previously which was caught by additional asserts.